MapboxNavigationUI/RouteMapViewController.swift

```diff
@@ -301,20 +347,22 @@ extension RouteMapViewController: MGLMapViewDelegate {
     }
     
     func mapView(_ mapView: MGLMapView, didChange mode: MGLUserTrackingMode, animated: Bool) {
-        if resetTrackingModeTimer != nil {
-            resetTrackingModeTimer.invalidate()
-        }
-        
-        if mode != .followWithCourse {
-            recenterButton.isHidden = false
-            startResetTrackingModeTimer()
-        } else {
-            recenterButton.isHidden = true
+        if !isInOverviewMode {
+            if resetTrackingModeTimer != nil {
```

Comment:
If you ever have to check for nil, that’s a sign you should make the variable optional (`?`) instead of implicitly unwrapped optional (`!`). Then you can simply call `resetTrackingModeTimer?.invalidate()` below.